Dockerfile.ci.broker

```diff
@@ -0,0 +1,15 @@
+# This assumes binaries are present, see COPY directive.
+
+FROM alpine AS chmodder
+ARG TARGETARCH
+COPY /artifacts/binaries-$TARGETARCH/broker /app/
+RUN chmod +x /app/*
+
+FROM gcr.io/distroless/cc
```

Comment:
The distroless image may not be the best choice because it has some high security issues. But I habe no better alternative.